airgun/views/all_hosts.py

```diff
@@ -605,3 +607,61 @@ class DisassociateHostsModal(PF5Modal):
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(self.title, exception=False) is not None
+
+class ChangeOrganizationLocationModal(PF5Modal):
+    """
+        This class represents 'Change organization/location' modal that is used to change org and location
+        for one or more hosts.
+    """
+    OUIA_ID = 'bulk-assign-taxonomy-modal'
+
+    # Organization section
+    organization_select = PF5Select(locator='.//div[@data-ouia-component-id="select-organization"]')
+    organization_fix_on_mismatch = PF5Radio(id='radio-org-fix-on-mismatch')
+    organization_fail_on_mismatch = PF5Radio(id='radio-org-fail-on-mismatch')
+
+    # Location section
+    location_select = PF5Select(locator='.//div[@data-ouia-component-id="select-location"]')
+    location_fix_on_mismatch = PF5Radio(id='radio-loc-fix-on-mismatch')
+    location_fail_on_mismatch = PF5Radio(id='radio-loc-fail-on-mismatch')
+
+    # Action buttons
+    save_button = Button(locator='.//button[@data-ouia-component-id="bulk-assign-taxonomy-modal-add-button"]')
+    cancel_button = Button(locator='.//button[@data-ouia-component-id="bulk-assign-taxonomy-modal-cancel-button"]')
+
+    # Alert messages
+    success_alert_title = PF5Alert(locator='.//h4[contains(@class, "-c-alert__title")]')
+
+    def fill(self, values):
+        """Fill the modal with organization, location, and option values.
+
+        Args:
+            values (dict): Dictionary containing:
+                - organization: Name of the organization to select
+                - location: Name of the location to select
+                - option: Either "Fix on mismatch" or "Fail on mismatch"
+        """
+        if 'organization' in values:
+            self.organization_select.fill(values['organization'])
+
+        if 'location' in values:
+            self.location_select.fill(values['location'])
+
+        if 'option' in values:
+            option = values['option']
+            if option == "Fix on mismatch":
+                self.organization_fix_on_mismatch.fill(True)
+                self.location_fix_on_mismatch.fill(True)
+            elif option == "Fail on mismatch":
+                self.organization_fail_on_mismatch.fill(True)
+                self.location_fail_on_mismatch.fill(True)
+            else:
+                raise ValueError(f"Invalid option: {option}. Must be 'Fix on mismatch' or 'Fail on mismatch'")
+
+    def save(self):
+        """Click the Save button to apply changes."""
+        self.save_button.click()
+
+    def cancel(self):
+        """Click the Cancel button to close modal without saving."""
+        self.cancel_button.click()
```

Comment:
@vijaysawant I would like to keep the logic of the added views out of the "views" files and keep it just to "entities" files as we do elsewhere.
You can get inspired i.e. here https://github.com/SatelliteQE/airgun/pull/1909.
There you can see clearly differentiated view definition in the "view" file and the logic using that view in the "entities" file.